scripts/LockedFarming.deploy.ts

```diff
@@ -15,8 +16,16 @@ async function main() {
     );
 
     const farming = await ethers.deployContract('SMD_v5', [LP, SFUND]);
+    await farming.waitForDeployment();
 
     console.log(`LockedFarming deployed to ${farming.address}`);
+
+    console.log('Waiting 20s, then verify contract...');
```

Comment:
We know that verification fails sometimes even if we wait those 20 seconds. Can we have some retry logic in place (up to a limited number of times)?